swagger.yaml

```
@@ -5568,5 +5832,82 @@ components:
             { "date": 1612553884, "calls": 89794, "endpoint": "block" },
           ]
 
+    nutlink_address:
+      type: object
+      properties:
+        metadata_url:
+          type: string
+          description: URL do specific metadata file
+          example: "https://nut.link/metadata.json"
+        metadata_hash:
+          type: string
+          description: Hash of the metadata file
+          example: "6bf124f217d0e5a0a8adb1dbd8540e1334280d49ab861127868339f43b3948af"
+        metadata:
+          type: object
+          description: The cached metadata of the `metadata_url` file.
+          additionalProperties: true
```

```suggestion
          additionalProperties: true
```

do we need this if we don't define any specific requirements on the metadata object?

---

My bad, definitely needed - without specifying the object structure it won't validate. So we either need this or explicitly define what's in metadata - I would vouch for the latter, but I'm not sure if this format is a standard.